Doc/library/sqlite3.rst

```diff
@@ -417,6 +417,18 @@ Module constants
    This flag may be combined with :const:`PARSE_COLNAMES` using the ``|``
    (bitwise or) operator.
 
+.. data:: SQLITE_OK
+          SQLITE_DENY
+          SQLITE_IGNORE
+
+   Flags used for return values from the callable passed as
+   the *authorizer_callback* argument of :meth:`Connection.set_authorizer`
+   to indicate whether:
```

Comment:
```suggestion
   Flags that should be returned by the *authorizer_callback* callable
   passed to :meth:`Connection.set_authorizer`, to indicate whether:
```

Reply:
Yeah, I really didn't like what I had here as it is long, dense and overcomplicated, but I was just too tired to think of something better, sorry. @ezio-melotti 's suggestion is a lot better, so I'll commit that. Thanks, Ezio!